### PR TITLE
Move imports out of local scope in unmerge task.

### PR DESCRIPTION
**Pull request by tkaemming**
This should be unnecessary after GH-5499.

This is essentially a revert of ac5dfebec12fb6aacde8edd5d88f6fbf9b28f181.

### CODE REVIEW

File: src/sentry/tasks/unmerge.py [hunk 1]
```diff
@@ -31,8 +39,6 @@ def fetch(*key):
 
 
 def get_caches():
-    from sentry.models import Environment, GroupRelease, Project, Release
-
     return {
         'Environment': cache(
```
Comment (tkaemming) [inline src/sentry/tasks/unmerge.py]: These could be modified back to direct model class references, I don't have a strong preference on whether that change should be made or not.

File: src/sentry/tasks/unmerge.py [hunk 2]
```diff
@@ -67,8 +73,6 @@ def merge_mappings(values):
 
 
 def get_culprit(event):
```
Comment (tkaemming) [inline src/sentry/tasks/unmerge.py]: This could go back to a `lambda` function.

File: src/sentry/tasks/unmerge.py [hunk 3]
```diff
@@ -90,8 +94,6 @@ def get_culprit(event):
 
 
 def get_event_score(caches, data, event):
-    from sentry.event_manager import ScoreClause
```
Comment (tkaemming) [inline src/sentry/tasks/unmerge.py]: This could go back to a `lambda` function.

**Review by JTCunning — APPROVED** [review]
Reacted with 👍/👎 for agreement.

Looks good, bonus points if this was ran in the same environment that caught the thread spawning debacle.

**Comment by tkaemming** [comment]
I made those changes. I didn't test this using the Trifecta of Doom because I didn't do the research on how to build a Docker image off of `HEAD` (but I should probably figure this out at some point) and am pretty confident the fix was isolated to the previous change and the imports here shouldn't have any effect. I'll test this as soon as it goes out though to verify.

**Pull request merged**